src/ipfs/schemas.ts

```diff
@@ -0,0 +1,101 @@
+import { z } from 'zod';
+
+const VERSION_1 = '1.0' as const;
+const VERSION_2 = '2.0' as const;
```

Comment:
I'd suggest including the zero in the name if you really want variables:
`const VERSION_1_0 = '1.0' as const;`